.idea/sonarIssues.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as above... I'm not sure this needs to be checked-in

Reply:
Fixed. Thanks.